zio-json/shared/src/main/scala/zio/json/decoder.scala

```diff
@@ -297,7 +297,7 @@ object JsonDecoder extends GeneratedTupleDecoders with DecoderLowPriority0 {
     final case class SumType(cons: String)       extends JsonError
   }
 
-  implicit val string: JsonDecoder[String] = new JsonDecoder[String] {
+  implicit def string: JsonDecoder[String] = new JsonDecoder[String] {
```

Comment:
I don’t think this change is necessary?

Reply:
Without it I was getting a null pointer exception in the new LowPriority trait I created